docs/docs-content/getting-started/additional-capabilities.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: "Palette Additional Capabilities"
+title: "Palette Additional Capabilities"
+description: "Learn more about Palette's Additional Capabilities."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["getting-started"]
+---
+
+Palette offers a range of additional capabilities designed to extend the functionality and versatility of Kubernetes
+workloads. These features comprise diverse deployment scenarios, ensuring organizations can manage their workloads
+across different environments. This can include data centers such as VMware vSphere and OpenStack, bare metal, or public
+cloud using your cloud accounts.
+
+Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), enforcing FIPS
```

Comment:
With edge, the mantra is:
many smaller clusters - i.e. the challenge is in the volume (count) of clusters, vs. the size of the clusters. Might want to add managing Edge locations at scale with potentially 1000s of sites.

Reply:
Nice! I rewrote the sentence.